server/concepts/clothingitem.ts

```diff
@@ -0,0 +1,97 @@
+import { Filter, ObjectId } from "mongodb";
+import DocCollection, { BaseDoc } from "../framework/doc";
+import { NotAllowedError, NotFoundError } from "./errors";
+
+export interface ClothingItemDoc extends BaseDoc {
+  name: String;
+  description: String;
+  owner: ObjectId;
+  borrower: ObjectId;
+  imageUrl: string;
+}
+
+export default class ClothingItemConcept {
+  public readonly clothingItems = new DocCollection<ClothingItemDoc>("clothingItems");
+
+  async create(owner: ObjectId, name: string, description: string, imageUrl: string) {
```

Comment:
Are you thinking of having the frontend convert the images to urls, or are you thinking of just doing that stuff later?

Reply:
Planning to have frontend convert the images to urls!